packages/forms-web-app/src/dynamic-forms/questions.js

```diff
@@ -674,39 +674,55 @@ exports.questions = {
 				'Select yes if the appeal site is in an area of outstanding natural beauty'
 			)
 		]
+	}),
+	designatedSitesCheck: new CheckboxQuestion({
+		title: 'Designated sites',
+		question: 'Is the development in, near or likely to affect any designated sites?',
+		fieldName: 'designated-sites-check',
+		url: 'designated-sites',
```

Comment:
You would also need to add a required validator to make sure the user has to choose an option.
If they don't choose any option, the error should be: Select a designated site, or select ‘No, it is not in, near or likely to affect any designated sites’

You will also need to use the ConditionalRequiredValidator to display an error in case they choose 'other' but don't enter anything in the text box.
In this case, the error should be: ConditionalRequiredValidator
(see copy deck: https://pins-ds.atlassian.net/wiki/spaces/AAPDS/pages/1295352047/CD15+-+Is+the+development+in+near+or+likely+to+affect+any+designated+sites+v13)